frontend/src/scenes/events/definitions/VolumeTable.scss

```diff
@@ -0,0 +1,47 @@
+.definition-description {
+    &.ant-input:placeholder-shown {
+        font-style: italic;
+    }
+}
+
+.panel-wrapper {
```

Comment:
Nit: This class name seems too generic and it's not scoped by another class, could cause unexpected behavior elsewhere.

Reply:
Late to the party and finally going through the code here... and yes, this should be fixed quickly, as it'll lead to a bunch of bugs otherwise 3 months from now :).

It's best to always wrap component-level `.scss` files inside some global "namespace" like:

```scss
.event-definition-volume-table {
    .definition-description { ... }
    .panel-wrapper { ... }
}
```

... and then add that class to the main div of your component.